README.md

```diff
@@ -101,9 +101,11 @@ We currently test Linux (Ubuntu and scratch), MacOS and Windows as packaged by
 * Interpreter
   * Linux is tested on amd64 (native) as well arm64 and riscv64 via emulation.
   * MacOS and Windows are only tested on amd64.
+  * NetBSD and OpenBSD are only tested on amd64.
```

Comment:
```suggestion
  * FreeBSD, NetBSD and OpenBSD are only tested on amd64.
```

Reply:
I'm sure it works, but are we testing the interpreter?
I mean, is that one of the tests?

Reply:
yeah spectests are running both interpreter and compiler when available

Reply:
Ah, I'm away from keyboard, I'll update when I'm back. Doing it here apparently borks the signoff?

Reply:
taking care of it!